front-end/src/views/profile-view/profile-view.component.tsx

```diff
@@ -2,10 +2,11 @@ import { FC } from "react";
 import { FriendsList } from "@domains/friends/components/friends-list";
 import { FriendsRequests } from "@domains/friends/components/friends-requests";
 import { FriendsSearch } from "@domains/friends/components/friends-search";
+import { FriendsEditSettingsButton } from "@domains/friends/components/friends-edit-settings-button";
 
 export const ProfileView: FC = () => (
   <>
-    {"what's up samson!"}
+    <FriendsEditSettingsButton />
```

Comment:
This component will now need to be called in a different place. Now that @Val-code-1 pushed in a placeholder UI, you'll have to pull down his merged changes. You can then call the `<FriendsEditSettingButton />` inside of his component! 😎 

Overall looks great so far!